docroot/profiles/custom/cgov_site/modules/custom/cgov_site_section/cgov_site_section.module

```diff
@@ -776,9 +776,13 @@ function cgov_site_section_entity_bundle_field_info_alter(&$fields, EntityTypeIn
   }
   if ($entity_type->id() == 'taxonomy_term'  && $bundle === 'cgov_site_sections') {
     if (isset($fields['parent'])) {
-      // Add custom constraint to this field
+      // Add custom constraint to this field.
```

Comment:
Please replace this comment with something more descriptive. A good comment should explain the _intent_ of the code.  In this case, what rule is being validated? That a parent exists? That there's only one?